flink-core/src/main/java/org/apache/flink/api/java/typeutils/PojoTypeInfo.java

```diff
@@ -419,4 +481,61 @@ public String toString() {
 			return "NamedFlatFieldDescriptor [name="+fieldName+" position="+getPosition()+" typeInfo="+getType()+"]";
 		}
 	}
+
+	public static String accessStringForField(Field f) {
+		String fieldName = f.getName();
+		if (Modifier.isPublic(f.getModifiers())) {
+			return fieldName;
+		}
+		String getterName = "get" + Character.toUpperCase(fieldName.charAt(0)) + fieldName.substring(1);
+		Class parentClazz = f.getDeclaringClass();
+		try {
+			parentClazz.getMethod(getterName, new Class[0]);
+		} catch (NoSuchMethodException e) {
+			// No getter, it might be a scala class.
```

Comment:
Is this possible at this point? The rules of being a POJO should not allow this, right?